File: umn.f90
function umn(m, n, x, y, L)
  implicit none
  real(8) umn
  real(8), parameter :: pi=3.141590d0
  integer(4) m, n
  real(8) x, y, L
  umn=cos(m*pi*x/L)*cos(n*pi*y/L)-cos(n*pi*x/L)*cos(m*pi*y/L)
end function umn

File: umn.c
double sin(double);
double cos(double);
double umn(int m, int n, double x, double y, double L) {
  static const double pi=3.14159265359; 
  return cos((double)m*pi*x/L)*cos((double)n*pi*y/L)
        -cos((double)n*pi*x/L)*cos((double)m*pi*y/L);
}
 
